app/controllers/solidus_bolt/accounts_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'base64'
+
+module SolidusBolt
+  class AccountsController < BaseController
+    def create
+      user = Spree::User.find_by(email: params[:email])
+      if user
+        render json: {}, status: :ok
```

Comment:
can we use instead use the following ?
```rb
render head :ok
```

Reply:
Maybe I'm wrong but if I remember correctly the controller can be refactored as:

```
def create
  user = Spree::User.find_by!(email: params[:email])
  
  render head: :ok
end
```

In that way, if the find_by! raise the `ActiveRecord::RecordNotFound` the controller should rescue it and return 404 with this [rescue_from](https://github.com/solidusio/solidus/blob/0d61702dacca5657f9c2cfb3cc48de8ab63a09f7/api/app/controllers/spree/api/base_controller.rb#L29)

Reply:
`render head :ok` returns ActionView::MissingTemplate error

between creating an empty file and writing `render json: {}, status: :ok` I think the second option is better

Reply:
modified in https://github.com/nebulab/solidus_bolt/commit/cbad318944309509197f3931924794104865de47

Reply:
"render head :ok" is a typo, it should be just "head :ok"

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/107/commits/f04c9f033e472d87940e5607bcd8eeb1d41349c5